charts/div-ccd-definitions/values.preview.template.yaml

```diff
@@ -0,0 +1,65 @@
+tags:
+  ccd-idam-pr: true
+
+nodejs:
+  # Don't modify below here
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+
+ccd:
+  ingressHost: ${SERVICE_FQDN}
+  ingressIP: ${INGRESS_IP}
+  consulIP: ${CONSUL_LB_IP}
+
+  idam-pr:
+    releaseNameOverride: ${SERVICE_NAME}-ccd-idam-pr
+    redirect_uris:
+      CCD:
+        - https://case-management-web-${SERVICE_FQDN}/oauth2redirect
+      CCD Admin:
+        - https://admin-web-${SERVICE_FQDN}/oauth2redirect
+
+  apiGateway:
+    s2sKey: ${API_GATEWAY_S2S_KEY}
+    idamClientSecret: ${API_GATEWAY_IDAM_SECRET}
+
+  userProfileApi:
+    authorisedServices: ccd_admin,ccd_data,ccd_definition,cmc_claim_store
+
+  dataStoreApi:
+    s2sKey: ${DATA_STORE_S2S_KEY}
+    s2sAuthorisedServices: cmc_claim_store,ccd_gw
+
+  definitionStoreApi:
+    s2sKey: ${DEFINITION_STORE_S2S_KEY}
+    s2sAuthorisedServices: ccd_admin,ccd_data,cmc_claim_store,ccd_gw
+
+  caseManagementWeb:
+    enabled: true # if you need access to the web ui then enable this, otherwise it won't be deployed
+    environment:
+      NODE_TLS_REJECT_UNAUTHORIZED: 0
+
+  adminWeb:
+    enabled: true # if you need access to the admin web ui then enable this, otherwise it won't be deployed
+    s2sKey: ${ADMIN_S2S_KEY}
+    idamClientSecret: ${ADMIN_WEB_IDAM_SECRET}
+    environment:
+      NODE_TLS_REJECT_UNAUTHORIZED: 0
+
+  importer:
```

Comment:
where is `definitionFilename` set? also I don't see `userRoles`

Reply:
We would have those once we decide to use the definition importer (https://github.com/hmcts/chart-ccd#importers), but at the moment we dont.
The problem is that we'd need the file built in this PR to be publicly avaliable somewhere, but we don't have that at the moment (unless you can think of a quick solution)
There is the other option of pulling it from a Docker image, but we'd have to publish that somewhere as well so for now it's manual to upload the file (roles are created via the functional test)

Reply:
Just had a thought, maybe I can use set `importer.definition.definitions` which accepts a URL. The URL will point to the sample app in the project, which will serve the generated file so that it can be imported - I'll try that

Reply:
Didn't work as the app is not accessible before the required services are up :/